contracts/test/helpers/ProposalTest.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.26;
+
+import {Test, console2} from "forge-std/Test.sol";
+import {GovernorCountingSimpleUpgradeable} from
+    "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorCountingSimpleUpgradeable.sol";
+import {CompoundGovernorTest} from "contracts/test/helpers/CompoundGovernorTest.sol";
+
+contract ProposalTest is CompoundGovernorTest {
+    struct ConstructedProposal {
```

Comment:
nit: does the word `Constructed` help disambiguate here, or can we drop it?

```suggestion
    struct Proposal {
```

Reply:
Done: 3fa8440bee64578a20f3bfa7a54b6496ade21310